Storage/Adapter/AwsS3AdapterConfiguration.php

```diff
@@ -66,6 +66,10 @@ public function buildForm(FormInterface $form, array $options = null)
                 'help_block' => 'Optional CNAME vs default AWS URL',
                 'label'=>'CNAME',
             ])
+            ->add('signature_version', TextType::class, [
```

Comment:
Perhaps we should use a ChoiceType instead, since there really only are certain allowed values here.

Reply:
Additionally, that allows us to default to `v4`